common/proc_allowing-external-access-knative-services-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+
+[id='allowing-external-access-knative-services-OCP-4x_{context}]
+= Allowing external access to Knative services on OCP 4.0
+
+Use a configured hostname to access Knative services on the OpenShift 4.0 cluster
+
+.Procedure
+
+. Updating Knative's config-domain ConfigMap and replacing "example.com" with your domain's suffix.
+
+NOTE: This will not be needed once the new operator capabilities are enabled to set the value automatically.
+
+.. Find out your cluster's domain suffix <YOUR CLUSTER SUFFIX> with the following command.
+
+   `oc get ingresses.config.openshift.io cluster -o jsonpath="{.spec.domain}"`
+   
+.. Edit Knative's config-domain ConfigMap and replace "example.com" with <YOUR CLUSTER SUFFIX>
```

Comment:
See comment from the first line of .Procedure - this doesn't need to be repeated, move that line to here?

Reply:
I think this is different to the first line; it's a subset of the procedure. Following what William wrote, I think it's outlined  as such:
1. Updating ...
1.1 Find your cluster...
1.2 Edit Knative...